docs/advanced-usage/compute-backends/kubernetes/index.md

```diff
@@ -108,3 +100,87 @@ You can set `kubernetes_memory_limit` in every `rule` under `resources`.
         "docker://python:2.7-slim"
     ...
 ```
+
+## Job timeout
+
+When the job exceeds the specified time limit, it will be terminated by Kubernetes and marked as failed.
+
+To set the job timeout, you can specify `kubernetes_job_timeout` in the specification of **each workflow step**.
+Time is measured in **seconds**.
+
+Read more about the expected values and timeout logic on [Kubernetes official documentation](https://kubernetes.io/docs/concepts/workloads/controllers/job/#job-termination-and-cleanup)
```

Comment:
The K8s documentation prefer the term "job time limit" and some such. They don't really use "timeout". I guess we can use the former as it should be clear, let's simply reformulate a bit:

```
Read more about the job run time deadline limits in the _Kubernetes official documentation_.
```

Note also the trailing dot.